lumen/ai/vector_store.py

```diff
@@ -572,7 +572,7 @@ async def query(
         text: str,
         top_k: int = 5,
         filters: dict | None = None,
-        threshold: float = 0.0,
+        threshold: float = -1.0,
```

Comment:
With -1 threshold, it actually includes all items, especially since NumpyEmbeddings does a poor job vector searching.